people/titles/engineer.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Infrastructure Engineer
+
+An Open Source Infrastructure (OSIE) focuses on infrastructure that supports interactive computing.
+It intersects job titles such as “dev-ops engineer”, “site reliability engineer”, “software engineer”, and “cloud engineer”.
+
+## Key responsibilities
+
+- Ensure the reliable operation of the 2i2c infrastructure (leveraging production-ready cloud-based tools such as JupyterHub, BinderHub and Dask).
+- Attend to support-facing tickets
```

Comment:
Something is nagging at me regarding "support-facing" being internal vernacular we use, but might not be very clear to someone reading this in order to info gather before making an application, for example. Can't quite express this nag though.